src/storage/leveldb/token_index.ts

```diff
@@ -49,9 +48,6 @@ export default class LevelTokenIndex implements IKVTokenIndex {
     this.tokenDB = db.sublevel<string, ITokenData>(TOKEN_PREFIX, { valueEncoding: 'json' });
     this.metadataDB = db.sublevel<string, ITokenMetadata>(META_PREFIX, { valueEncoding: 'json' });
     this.registeredDB = db.sublevel<string, ITokenData>(REGISTER_PREFIX, { valueEncoding: 'json' });
-
-    // Add HTR to the database
-    this.saveToken(HATHOR_TOKEN_CONFIG);
```

Comment:
How are we handling the default token on storage now? We need to save the native token on it, don't we?

Reply:
Yes, the storage will start empty and the wallet will need to add the token(s) it wants the wallet to start with.